### ISSUE TITLE
DTR concern - AUG 21 - SEPT 5

### ISSUE THREAD

**Issue by acmiras** [vasavah/vci-kanaka_auntie-annes-testing-phase#76]
![image](https://user-images.githubusercontent.com/60170851/95693749-8dc7b000-0c60-11eb-805b-0caf1ebb5da5.png)

![image](https://user-images.githubusercontent.com/60170851/95693752-96b88180-0c60-11eb-9e47-928743521a83.png)

![image](https://user-images.githubusercontent.com/60170851/95693759-9fa95300-0c60-11eb-80f3-727b23da4072.png)

![image](https://user-images.githubusercontent.com/60170851/95693770-a9cb5180-0c60-11eb-85f8-af7e8a9cd76c.png)

ETO PO YUNG SA KIOSK

IN SUMMARY

OT: 3HOURS + 5.50 HOURS +2.5 HOURS + 1.5 HOURS = 12.5 HOURS

LATE : 30 MINS + 50 MINS = 80 MINS

OVERBREAK : 0.50 + 0.17 + 0.25 = 0.92

UNDERTIME : 4 HOURS + 1 HOURS = 5 HOURS

**Comment by acmiras** [comment]
![image](https://user-images.githubusercontent.com/60170851/95693896-6de4bc00-0c61-11eb-995f-0557921a2c5f.png)


![image](https://user-images.githubusercontent.com/60170851/95693907-8359e600-0c61-11eb-8e6e-42db0c173aaf.png)


![image](https://user-images.githubusercontent.com/60170851/95693909-8a80f400-0c61-11eb-9de6-4fc9e096316c.png)


SIR KULANG YUNG DATA AT MALI MALI YUNG PUWESTO NUNG NUMBERS TAPOS KULANG YUNG DATA PO

**Comment by acmiras** [comment]
ETO PO YUNG TIME AND ATTENDANCE REPORT

![image](https://user-images.githubusercontent.com/60170851/95694192-2e1ed400-0c63-11eb-91a7-e928f14217c4.png)


9 DAYS ?

HND PO BAYAD YUNG DALAWANG LEGAL HOLIDAY

AUG 21 ANG AUG 31 .. 

TAPOS YUNG LATE AND UNDERTIME NAKA PER HOUR PO SILA DI BA HND SILA NAKA PER MINUTE PUWEDE BANG I PER MINUTES SILA TAPOS YUNG OVERBREAK WALA SYANG SARILING COLOMN 

LATE AT UNDERTIME LANG ANG MERON

![image](https://user-images.githubusercontent.com/60170851/95694261-a4233b00-0c63-11eb-8a3f-f76206760dc6.png)


YUNG OVERTIME NA 1.5 KULANG DIN PO ITO

![image](https://user-images.githubusercontent.com/60170851/95694284-d16fe900-0c63-11eb-884f-5515035a0dff.png)


Spcl NWH   NDOT
--
 - ANU PO ITO? TAPOS MAY NAKA LAGAY NA 12. 
--

**Comment by VCI-Jojo** [comment]
Closing this ticket as validated via G Meets for validation of the DTR with TAR.
